### PR TITLE
completed April 24th

### PR DESCRIPTION
**Pull request by PendarShams**

### CODE REVIEW

File: src/PenTasks/Week03/ReversedNegativeNumbers.java
Comment (cbanavi) [inline src/PenTasks/Week03/ReversedNegativeNumbers.java:1]: You can keep this negative but lgtm

File: src/PenTasks/Week03/PrimeNumber.java
Comment (cbanavi) [inline src/PenTasks/Week03/PrimeNumber.java:1]: This is returning me 5 is not a prime number

**Review by cbanavi — CHANGES_REQUESTED** [review]

File: src/PenTasks/Week03/PrimeNumber.java
Comment (cbanavi) [reply src/PenTasks/Week03/PrimeNumber.java:1]: Now it's telling me basically all numbers are prime

**Review by cbanavi — CHANGES_REQUESTED** [review]

File: src/PenTasks/Week03/PrimeNumber.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    public static void main(String[] args) {
+        System.out.println(primeFinder(5));
+    }
+
+    public static boolean primeFinder(int num) {
+        if (num <= 1) {
+            System.out.println("Please choose a number greater than 1!");
+            return false;
+        } else if (num % num == 0) {
```
[the remark this answers is not in the document]
Comment (PendarShams) [reply src/PenTasks/Week03/PrimeNumber.java]: corrected

File: src/PenTasks/Week03/ArmstrongNumbers.java
Comment (cbanavi) [inline src/PenTasks/Week03/ArmstrongNumbers.java:1]: This isn't returning true or false, just the print statement.

File: src/PenTasks/Week03/PrimeNumber.java
Comment (cbanavi) [inline src/PenTasks/Week03/PrimeNumber.java:1]: lgtm

File: src/PenTasks/Week03/ReversedNegativeNumbers.java
Comment (cbanavi) [inline src/PenTasks/Week03/ReversedNegativeNumbers.java:1]: lgtm

**Review by cbanavi — CHANGES_REQUESTED** [review]
Minor changes

**Review by cbanavi — APPROVED** [review]
lgtm

**Pull request merged**